dali/dfu/dfuwu.cpp

```diff
@@ -2972,6 +3135,34 @@ class CDFUWorkUnitFactory : implements IDFUWorkUnitFactory, implements ISDSSubsc
         // created time stamp? TBD
         return ret;
     }
+
+    IDFUWorkUnit * createPublisherWorkUnit() override
+    {
+        StringBuffer wuid;
+        newWUID(wuid, 'P');
```

Comment:
question: does this new dfu workunit name format pose any issues for esp/eclwatch services?
(it really shouldn't - as they should be using interfaces to get results etc.)

Reply:
It doesn't break EclWatch for the old workunits.  

I was waiting to see if this way of doing things got approved and then I need to open a jira about improving the way it interacts with the new ones.

In the future opening a publisher workunit should open 2 child tabs:
   1. A list of the child dfu workunits (same control as listing DFU workunits now).
   2. The publisher workunit xml.